README.md

```diff
@@ -6,39 +6,165 @@ Please edit this README.md file with your answers to the worksheet questions.
 
 ### a
 
+| A | B	| C	| A AND B	|	A AND B AND NOT C |
+|---|---|---|---|---|
+| TRUE	| TRUE | TRUE	| TRUE	| FALSE |
+| TRUE	| TRUE | FALSE | TRUE	| TRUE |
+| TRUE	| FALSE	| TRUE	| FALSE	| FALSE |
+| TRUE	| FALSE	| FALSE	| FALSE	| FALSE |
+| FALSE	| TRUE	| TRUE	| FALSE	| FALSE |
+| FALSE	| TRUE	| FALSE	| FALSE	| FALSE |
+| FALSE	| FALSE	| TRUE	| FALSE	| FALSE |
+| FALSE	| FALSE	| FALSE	| FALSE	| FALSE |
+
 ### b
 
+| A	| B	| C	| B AND NOT C	| A AND NOT (B AND NOT C) |
+|---|---|---|---|---|
+| TRUE	| TRUE	| TRUE	| FALSE		| TRUE |
+| TRUE	| TRUE	| FALSE	| TRUE		| FALSE |
+| TRUE	| FALSE	| TRUE	| FALSE		| TRUE |
+| TRUE	| FALSE	| FALSE	| FALSE		| TRUE |
+| FALSE	| TRUE	| TRUE	| FALSE		| FALSE |
+| FALSE	| TRUE	| FALSE	| TRUE		| FALSE |
+| FALSE	| FALSE	| TRUE	| FALSE		| FALSE |
+| FALSE	| FALSE	| FALSE	| FALSE		| FALSE |
+
 ### c
 
+| A	| B	| C	| A OR NOT B	| A OR C		| (A OR NOT B) AND (A OR C) |
+|---|---|---|---|---|---|
+| TRUE	| TRUE	| TRUE	| TRUE		| TRUE		| TRUE |
+| TRUE	| TRUE	| FALSE	| TRUE		| TRUE		| TRUE |
+| TRUE	| FALSE	| TRUE	| TRUE		| TRUE		| TRUE |
+| TRUE	| FALSE	| FALSE	| TRUE		| TRUE		| TRUE |
+| FALSE	| TRUE	| TRUE	| FALSE		| TRUE		| FALSE |
+| FALSE	| TRUE	| FALSE	| FALSE		| FALSE		| FALSE |
+| FALSE	| FALSE	| TRUE	| TRUE		| TRUE		| TRUE |
+| FALSE	| FALSE	| FALSE	| TRUE		| FALSE		| FALSE |
+
+
 ### d
 
+| A	| B	| C	| D	| B OR NOT C	| NOT A AND D	| A AND NOT (B OR NOT C) AND (NOT A AND D)| 
+|---|---|---|---|---|---|---|
+| TRUE	| TRUE	| TRUE	| TRUE	| TRUE		| FALSE		| FALSE| 
+| TRUE	| TRUE	| TRUE	| FALSE	| TRUE		| TRUE		| FALSE| 
+| TRUE	| TRUE	| FALSE	| TRUE	| TRUE		| FALSE		| FALSE| 
+| TRUE	| TRUE	| FALSE	| FALSE	| TRUE		| TRUE		| FALSE| 
+| TRUE	| FALSE	| TRUE	| TRUE	| FALSE		| FALSE		| FALSE| 
+| TRUE	| FALSE	| TRUE	| FALSE	| FALSE		| TRUE		| TRUE| 
+| TRUE	| FALSE	| FALSE	| TRUE	| TRUE		| FALSE		| FALSE| 
+| TRUE	| FALSE	| FALSE	| FALSE	| TRUE		| TRUE		| FALSE| 
+| FALSE	| TRUE	| TRUE	| TRUE	| TRUE		| TRUE		| FALSE| 
+| FALSE	| TRUE	| TRUE	| FALSE	| TRUE		| FALSE		| FALSE| 
+| FALSE	| TRUE	| FALSE	| TRUE	| TRUE		| TRUE		| FALSE| 
+| FALSE	| TRUE	| FALSE	| FALSE	| TRUE		| FALSE		| FALSE| 
+| FALSE	| FALSE	| TRUE	| TRUE	| FALSE		| TRUE		| FALSE| 
+| FALSE	| FALSE	| TRUE	| FALSE	| FALSE		| FALSE		| FALSE| 
+| FALSE	| FALSE	| FALSE	| TRUE	| TRUE		| TRUE		| FALSE| 
+| FALSE	| FALSE	| FALSE	| FALSE	| TRUE		| FALSE		| FALSE| 
+
 ## Question 2
```

Comment:
Using a circle to denote a NOT gate is not a standard symbol (or at least I've never seen it before) -- better to use the standard NAND/NOR/NOT gate symbols as shown in class.